ui/tests/unit/queries.spec.js

```diff
@@ -335,60 +335,56 @@ describe("Search", () => {
-    ["test"],
-    ["term", "test"],
-    ["term", " test "],
-    ["lastUpdated", "<", "2000", "term", "test"],
-    ["term", "test", "lastUpdated", "<", "2000"]
-  ])("Given %p parses term", async (...args) => {
+    ["test", "test"],
+    ["two words", "two words"],
+    ["test lastUpdated:>2020", "test"],
+    ["lastUpdated:2019..2020 test", "test"]
+  ])("Given %p parses term", async (value, expected) => {
     const wrapper = mountFunction({
-      data: () => ({ value: args })
+      data: () => ({ inputValue: value })
     });
 
-    const input = wrapper.find(".v-select__selections > input");
-    await input.trigger("change");
+    const button = wrapper.find("button.mdi-magnify");
+    await button.trigger("click");
 
-    expect(wrapper.vm.searchFilters.term).toBe("test");
+    expect(wrapper.vm.filters.term).toBe(expected);
   });
 
   test.each([
-    [["lastUpdated", "<", "2000"], "<2000-01-01T00:00:00.000Z"],
-    [["lastUpdated", "<", "2000-10-01"], "<2000-10-01T00:00:00.000Z"],
-    [["lastUpdated", "<=", "2000-08-10"], "<=2000-08-10T00:00:00.000Z"],
-    [["lastUpdated", ">", "2000"], ">2000-01-01T00:00:00.000Z"],
-    [["lastUpdated", ">=", "2000-02-03"], ">=2000-02-03T00:00:00.000Z"],
+    ["lastUpdated:<2000", "<2000-01-01T00:00:00.000Z"],
+    ["lastUpdated:<=2000-08-10", "<=2000-08-10T00:00:00.000Z"],
+    ["lastUpdated:>2000", ">2000-01-01T00:00:00.000Z"],
+    ["lastUpdated:>=2000-02-03", ">=2000-02-03T00:00:00.000Z"],
     [
-      ["lastUpdated", "range", "2000..2001"],
+      "lastUpdated:2000..2001",
       "2000-01-01T00:00:00.000Z..2001-01-01T00:00:00.000Z"
     ],
-    [["term", "test", "lastUpdated", "<", "2000"], "<2000-01-01T00:00:00.000Z"]
-  ])("Given %p parses lastUpdated filter", async (values, expected) => {
+    ["test lastUpdated:<2000", "<2000-01-01T00:00:00.000Z"]
+  ])("Given %p parses lastUpdated filter", async (value, expected) => {
     const wrapper = mountFunction({
-      data: () => ({ value: values })
+      data: () => ({ inputValue: value })
     });
 
-    const input = wrapper.find(".v-select__selections > input");
-    await input.trigger("change");
+    const button = wrapper.find("button.mdi-magnify");
+    await button.trigger("click");
 
-    expect(wrapper.vm.searchFilters.lastUpdated).toBe(expected);
+    expect(wrapper.vm.filters.lastUpdated).toBe(expected);
   });
 
   test.each([
-    ["lastUpdated", "abc"],
-    ["lastUpdated", "<", "abc"],
-    ["lastUpdated", "<", "2000-23-01"],
-    ["lastUpdated", ">=", "2000-01-49"],
-    ["lastUpdated", ">", "@"],
-    ["lastUpdated", "range", "2000"],
-    ["lastUpdated", "range", "2000-2001"],
-    ["lastUpdated", "range", "2001-2000"]
-  ])("Given an invalid value renders an error", async (...args) => {
+    "lastUpdated:abc",
+    "lastUpdated:<abc",
+    "lastUpdated:<2000-23-01",
+    "lastUpdated:>=2000-01-49",
+    "lastUpdated:>@",
+    "lastUpdated:2000-2001",
+  ])("Given an invalid value renders an error", async (value) => {
```

Comment:
I think there must be a test case where `lastUpdated` has two operators. It should return an error too. For example `lastUpdated:>=2000-01-01<=2000-01-01` or `lastUpdated:===2000-01-01`.